keyboards/plexus75/plexus75.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
```

Comment:
Need a license header here, please.

Reply:
added

Reply:
`keyboards/plexus75/plexus75.h` still doesn't have a license header.

Reply:
sorry, it should be there now